ecosystem/typescript/sdk_v2/src/types/index.ts

```diff
@@ -852,6 +858,16 @@ export enum AuthenticationKeyScheme {
    * For MultiEd25519PublicKey
    */
   MultiEd25519 = 1,
+  /**
+   * For Secp256k1 ecdsa
+   */
+  Secp256k1Ecdsa = 2,
+}
+
+/**
+ * Scheme used for deriving account addresses from other data
```

Comment:
let's add a link here also. let's also mention these enums are represented in the Rust code